apps/frontend/src/libs/components/button/button.tsx

```diff
@@ -3,15 +3,17 @@ import { getValidClassNames } from "~/libs/helpers/helpers.js";
 import styles from "./styles.module.css";
 
 type Properties = {
+	icon?: React.ReactNode;
 	isFluid?: boolean;
 	isPrimary?: boolean;
-	label: string;
+	label?: string;
```

Comment:
```suggestion
	label: string;
```
Label should be present. Add `hasVisuallyHiddenLabel` prop and if it is true, hide the label using `visually-hidden` class